### PR TITLE
Add diff b/w Latch and Flip flop

### PR DESCRIPTION
**Pull request by Ananthavijay**
Fixes https://github.com/CircuitVerse/Interactive-Book/issues/39
[GCI TASK](https://codein.withgoogle.com/dashboard/task-instances/6246207073026048/)

- Added differences between Flip flop and latch under Flip flop.

## Screenshot :

Screenshot (134)

### CODE REVIEW

**Comment by Ananthavijay** [comment]
@eKuG Please review.

**Comment by eKuG** [comment]
Looks good

File: docs/flipflop/flipflops.md [hunk 1]
```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
 
 In second module, we can directly implement the flip-flop, which is edge sensitive. In this module, let us discuss the following flip-flops using second method.
 
+## Difference between Latch and Flip Flop
```
Comment (JoshHeng) [inline docs/flipflop/flipflops.md]: Change to `Differences Between Latches and Flip-Flops`

File: docs/flipflop/flipflops.md [hunk 2]
```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
...
+
+
+| Latch       | Flip flop     |
```
Comment (JoshHeng) [inline docs/flipflop/flipflops.md]: Change to `Latches` and `Flip-Flops`

File: docs/flipflop/flipflops.md [hunk 3]
```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
...
+|:------------|:--------------|
+| Latches are level triggered | Flip flops are edge triggered |
+| Latch is a asynchronous device | Flip flop is a synchronous device |
+| Latch doesn't contain clock signal | Flip flop contains a clock signal |
+| Latch will change its state as long it is enabled | Flip flop will change its state only during a given clock cycle |
+| Latch consumes less power | Flip flop consumes more power |
+| Latches are faster | Flip flops are slower |
+| Examples : D-Latch, T-Latch | Examples : D-Flip flop, T-Flop flop |
```
Comment (JoshHeng) [inline docs/flipflop/flipflops.md]: No need for a space after `Examples` - just do `Examples: D-Latch & T-Latch`

File: docs/flipflop/flipflops.md [hunk 4]
```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
 
 In second module, we can directly implement the flip-flop, which is edge sensitive. In this module, let us discuss the following flip-flops using second method.
 
+## Difference between Latch and Flip Flop
+
+
+| Latch       | Flip flop     |
+|:------------|:--------------|
+| Latches are level triggered | Flip flops are edge triggered |
```
Comment (JoshHeng) [inline docs/flipflop/flipflops.md]: No need for `Latches are` and `Flip flops are`

You can just use `Level triggered | Edge triggered` as the table headings explain the rest. This also applies to the other points

File: docs/flipflop/flipflops.md [hunk 5]
```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
...
 
+## Differences between Latches and Flip-Flops
+
+
+| Latches       | Flip-Flops     |
+|:------------|:--------------|
+| Level triggered | Edge triggered |
+| Asynchronous device | Synchronous device |
+| Doesn't contain clock signal | Contains a clock signal |
+| It will change its state as long it is enabled | It will change its state only during a given clock cycle |
```
Comment (Shivansh2407) [inline docs/flipflop/flipflops.md:26]: It should be `as long as`

File: docs/flipflop/flipflops.md [hunk 6]
```diff
@@ -15,6 +15,18 @@ In first method, cascade two latches in such a way that the first latch is enabl
...
+| Consumes less power | Consumes more power |
+| Faster | Slower |
```
Comment (Shivansh2407) [inline docs/flipflop/flipflops.md:28]: Can you please mention Faster and Slower in what reference?

Comment (Ananthavijay) [reply docs/flipflop/flipflops.md:28]: Latches work faster than flip-flop. I dont think that it needs to be mentioned

File: docs/flipflop/flipflops.md (hunk 5 above)
Comment (Ananthavijay) [reply docs/flipflop/flipflops.md]: Done

**Comment by Ananthavijay** [comment]
@Shivansh2407 Please have a look

**Pull request merged**